### DISCUSSION TITLE
help someone is in my account

### DISCUSSION THREAD

**Discussion by 1alexbc1** [community/community#134111] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### GitHub Feature Area

Discussions

### Body

Please can someone help me, I'm waking up, I see that everything has changed, I have also seen other bank account numbers on some platforms, Google and Microsoft have also reported to me that hackers and/or several bugs have been posted, I would have liked some help with this.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @1alexbc1 , Thanks for reaching out! 

We're sorry to hear you're experiencing this issue.

GitHub is a software hosting service, we host millions of projects, so probably every phone and computer on the planet has software on it that is also hosted here.

We're not able to provide support or answers for those individual projects. GitHub Support can only provide support for GitHub itself and as result we'll be closing and locking this discussion.

I'm sorry we can't help here but we'd recommend contacting a local IT specialist for assistance if you think your devices are affected by viruses or malware. They should be able to help you with these applications!

_Please note for future posts:  The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._
